### PR TITLE
Add NI-DMM configure system tests and remove configure functions that are already attributes.

### PR DESCRIPTION
**Pull request by Fladolcetta**
[x ] This contribution adheres to [CONTRIBUTING.md]

### What does this Pull Request accomplish?

Adds additional system tests for configure functions with multiple parameters.
Updates functions addon to fix enums for the above functions.
Removes configure functions that have a single parameter.

### What testing has been done?

Built and ran unit tests and system tests

### CODE REVIEW

File: src/nidmm/system_tests/test_system_nidmm.py [hunk 1]
```diff
@@ -197,3 +197,35 @@ def test_self_cal(session):
         session.self_cal()
     except nidmm.Error as e:
         assert False
+
+
+def test_configure_functions(session):
```
Comment (texasaggie97) [inline src/nidmm/system_tests/test_system_nidmm.py]: Should these be broken in to separate functions? I'm not sure what the "Pythonic" way would be...

File: src/nidmm/metadata/functions_addon.py [hunk 2]
```diff
@@ -34,31 +34,34 @@
     'GetCalCount':                     { 'codegen_method': 'no',       },  # Calibration function not exposed in Python API
     'FormatMeasAbsolute':              { 'codegen_method': 'no',       },  # Utility function for C customers
     'IsUnderRange':                    { 'codegen_method': 'no',       },  # Utility function for C customers
-    'IsOverRange':                     { 'codegen_method': 'no',       },  # Utility function for C customers
+    'ConfigureThermistorType':         { 'codegen_method': 'no',       },  # Utility function for C customers
```
Comment (marcoskirsch) [inline src/nidmm/metadata/functions_addon.py]: inaccurate comment

File: src/nidmm/metadata/functions_addon.py [hunk 3]
```diff
@@ -34,31 +34,34 @@
...
+    'ConfigureTransducerType':         { 'codegen_method': 'no',       },  # Attribute already available
```
Comment (marcoskirsch) [inline src/nidmm/metadata/functions_addon.py]: use the same wording we used in NI-DCPower's addon.py

File: src/nidmm/system_tests/test_system_nidmm.py (hunk 1 above)
Comment (marcoskirsch) [reply src/nidmm/system_tests/test_system_nidmm.py]: separate

File: src/nidmm/metadata/functions_addon.py (hunk 2 above)
Comment (Fladolcetta) [reply src/nidmm/metadata/functions_addon.py]: Fixed

File: src/nidmm/metadata/functions_addon.py (hunk 3 above)
Comment (Fladolcetta) [reply src/nidmm/metadata/functions_addon.py]: Fixed

**Pull request merged**